scarb/src/compiler/plugin/proc_macro/v2/host/mod.rs

```
@@ -214,6 +214,12 @@ impl MacroPlugin for ProcMacroHostPlugin {
 
         // Expand module-level inline macro.
         if let ast::ModuleItem::InlineMacro(inline_macro) = &item_ast
+            && inline_macro
+                .path(db)
+                .segments(db)
+                .elements(db)
+                .exactly_one()
+                .is_ok()
```

Rather than skipping it, should we perhaps handle this explicitly, and return error similar to one that Cairo emits?
```
error: Inline macro `not::a::path::some` not found.
```